packages/lib/src/vis/shared/VisCanvas.tsx

```diff
@@ -49,8 +49,10 @@ function VisCanvas(props: PropsWithChildren<Props>) {
       })
     : NO_OFFSETS;
 
-  const [svgOverlay, setSvgOverlay] = useState<SVGSVGElement>();
-  const [floatingToolbar, setFloatingToolbar] = useState<HTMLDivElement>();
+  const [svgOverlay, setSvgOverlay] = useState<SVGSVGElement | null>(null);
+  const [floatingToolbar, setFloatingToolbar] = useState<HTMLDivElement | null>(
+    null,
+  );
```

Comment:
It's simpler for `svgOverlay` and `floatingToolbar` to never be `undefined` (see below). Also, I find that `null` better conveys the fact that the containers are there but we just can't get their refs synchronously.